components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -110,6 +114,12 @@ public class ServerConfigManagementService {
 
     private static final Log log = LogFactory.getLog(ServerConfigManagementService.class);
 
+    private static final List<String> FEDERATED_AUTHENTICATORS_TO_UPDATE = new ArrayList<>(Arrays.asList(
```

Comment:
Though we use federated authenticators to represent the configs in the resident IDP, naming the constant like this adds confusions IMO.

Reply:
Removed. Logic is updated to only change the required federated authenticator